src/main.js

```diff
@@ -0,0 +1,19 @@
+import FilterView from './view/filter-view.js';
+import SortListView from './view/sort-list-view.js';
+import TripInfoView from './view/trip-info-view';
+import {render} from './render.js';
+import {RenderPosition} from './render.js';
+
+import RenderComponentsPresenter from './presenter/render-components-presenter';
```

Comment:
Название максимально абстрактное, так любой презентер можно назвать. Мне нужно не залезая внутрь понять примерно что он делает.
```suggestion
import TripPresenter from './presenter/render-components-presenter';
```